Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -952,6 +955,7 @@ def start_endpoint_security_assessment_command(client: Client, template_id: str,
     params = {
         'templateID': template_id,
         'agentName': agent_name,
+        'agentProfileName': agent_Profile_Name,
```

Comment:
Since it is not mandatory, is the API ready to accept it as None or something like that?

Reply:
same as above